modules/varnish/templates/default.vcl

```diff
@@ -165,6 +165,7 @@ sub mw_request {
 		req.http.Host == "sslrequest.miraheze.org"
 	) {
 		set req.backend_hint = mwtask111;
+		set req.backend_hint = puppet111;

```

Comment:
Why do we have it set twice and not once?

Reply:
Oh, can it be done with commas? I don't want to remove mwtask111 until the switch is actually done